colors/whitescale.vim

```diff
@@ -9,7 +9,7 @@ endif
 let g:colors_name = "whitescale"
 
 if &background == "light"
-  hi ColorColumn term=NONE cterm=NONE gui=NONE ctermfg=white ctermbg=234 guifg=white guibg=#1C1C1C
+  hi ColorColumn term=NONE cterm=NONE gui=NONE ctermfg=white ctermbg=lightgray guifg=white guibg=lightgray
```

Comment:
I'm unsure about compatibility here, I'd use the 256-based colors for `cterm` at least.

Reply:
I was under the impression `lightgray` was one of those but honestly I know next to nothing about all this 😞.